pyfr/solvers/navstokes/kernels/bcs/slp-adia-wall.mako

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+
+<%include file='pyfr.solvers.euler.kernels.rsolvers.${rsolver}'/>
+<%include file='pyfr.solvers.navstokes.kernels.bcs.common'/>
+<%include file='pyfr.solvers.navstokes.kernels.flux'/>
+
+<% tau = c['ldg-tau'] %>
```

Comment:
Defined but never used.  Unsure if we should be including a penalty term or not.